modules/osd-intro.adoc

```diff
@@ -5,7 +5,7 @@
 [id="osd-intro_{context}"]
 = An overview of {product-title}
 
-{product-title} is professionally managed by Red Hat and hosted on {AWS} or {GCP}. Each {product-title} cluster comes with a fully managed link:https://access.redhat.com/documentation/en-us/openshift_container_platform/{product-version}/html/architecture/control-plane[control plane] (Control and Infrastructure nodes), application nodes, installation and management by Red Hat Site Reliability Engineers (SRE), premium Red Hat Support, and cluster services such as logging, metrics, monitoring, notifications portal, and a cluster portal.
+{product-title} is professionally managed by Red Hat and hosted on {AWS} or {GCP}. Each {product-title} cluster comes with a fully managed link:https://access.redhat.com/documentation/en-us/openshift_container_platform/{ocp-version}/html/architecture/control-plane[control plane] (Control and Infrastructure nodes), application nodes, installation and management by Red Hat Site Reliability Engineers (SRE), premium Red Hat Support, and cluster services such as logging, metrics, monitoring, notifications portal, and a cluster portal.
```

Comment:
```suggestion
Red Hat professionally manages {product title} for you on either {AWS} or {GCP}. Each {product-title} cluster you use includes a fully managed link:https://access.redhat.com/documentation/en-us/openshift_container_platform/{ocp-version}/html/architecture/control-plane[control plane] (Control and Infrastructure nodes), application nodes, and Red Hat's Site Reliability Engineers (SRE) handle installation and management. You also get premium support from Red Hat and key cluster services such as logging, metrics, monitoring, notifications portal, and a cluster portal.
```

Reply:
:_mod-docs-content-type: variable is missing?

Reply:
Thank you, @agantony. Great suggestions. 
Because this is a key product description, seemingly minor changes might require review by the product owner.
I prefer to limit this PR to fixing only the link issue.